cache/src/main/java/com/uber/buckcache/resources/buckcache/BuckCacheResource.java

```diff
@@ -77,6 +78,29 @@ public String getSummary() throws Exception {
     }
   }
 
+  /**
+   * Used for client side to measure download speed.
+   * @param size
+   * @return
+   * @throws Exception
+   */
+  @GET
+  @Path("dummy")
+  @Produces(MediaType.APPLICATION_OCTET_STREAM)
+  public String getDummyArtifact() throws Exception {
+    Random rand = new Random();
+    rand.setSeed(System.currentTimeMillis());
+    try {
+      // 100Kb
+      byte[] bytes = new byte[100000];
+      rand.nextBytes(bytes);
+      return new String(bytes);
```

Comment:
Why don't we simply return `byes[]` from this function?
I think converting to `String` is what is causing it to generate variable length artifact every time.

[docs](https://docs.oracle.com/javase/7/docs/api/java/lang/String.html#String(byte[])) say:
> public String(byte[] bytes)
Constructs a new String by decoding the specified array of bytes using the platform's default charset. The length of the new String is a function of the charset, and hence may not be equal to the length of the byte array.
The behavior of this constructor when the given bytes are not valid in the default charset is unspecified. The CharsetDecoder class should be used when more control over the decoding process is required.